src/core/sys/posix/netinet/in_.d

```diff
@@ -972,23 +972,4 @@ else version( Android )
 IPPROTO_RAW
 */
 
-version( linux )
-{
-    enum uint IPPROTO_RAW = 255;
-}
-else version( OSX )
-{
-    enum uint IPPROTO_RAW = 255;
-}
-else version( FreeBSD )
-{
-    enum uint IPPROTO_RAW = 255;
-}
-else version( Solaris )
-{
-    enum uint IPPROTO_RAW = 255;
-}
-else version( Android )
-{
-    enum uint IPPROTO_RAW = 255;
-}
+enum uint IPPROTO_RAW = 255;
```

Comment:
[Must be defined in POSIX standard, but actual value is not specified](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/netinet_in.h.html).